sendrecv/gst-rust/src/main.rs

```diff
@@ -0,0 +1,480 @@
+extern crate clap;
+extern crate glib;
+extern crate gstreamer as gst;
+extern crate gstreamer_sdp;
+extern crate gstreamer_sdp_sys;
+extern crate gstreamer_webrtc;
+extern crate rand;
+extern crate ws;
+#[macro_use]
+extern crate serde_json;
+
+use gst::prelude::*;
+use gst::{BinExt, ElementExt};
```

Comment:
This line should be unneeded, `gst::prelude::*` already imports all the ext traits into the scope